lib/widget/button.dart

```diff
@@ -29,6 +31,7 @@ class Button extends StatefulWidget with Invokable, HasController<ButtonControll
   Map<String, Function> setters() {
     return {
       'label': (value) => _controller.label = Utils.getString(value, fallback: ''),
+      'labelWidget': (value) => _controller.labelWidget = value,
```

Comment:
we do this everywhere for widget but something to think about: we have to assume what the user enters here will resolve to a valid widget later (or else we show 'invalid widget' text). Probably need a `Utils.getWidget(value)` here where we check against the list of valid widgets, plus any user-defined custom widget.

Reply:
The check makes sense but how could we handle it differently if it doesn't resolve to a valid widget? For example, fail silently or return a default widget?

Reply:
Got it. Shall I create a new issue for the widget util validation or is there one already?

Reply:
https://github.com/orgs/EnsembleUI/projects/2/views/1?pane=issue&itemId=23052103